dialogue-core/src/main/java/com/palantir/dialogue/core/DeprecationWarningChannel.java

```diff
@@ -56,7 +57,7 @@
                 delegate.execute(endpoint, request), DialogueFutures.onSuccess(response -> {
                     if (response != null) {
                         response.getFirstHeader("deprecation").ifPresent(deprecated -> {
-                            metrics.deprecations(endpoint.serviceName()).mark();
```

Comment:
I think `endpoint.serviceName()` makes the most sense here since deprecation is defined in conjure with the endpoint itself.